koap/src/commonTest/kotlin/ByteArrayReaderTest.kt

```diff
@@ -10,6 +10,7 @@ import kotlin.test.assertTrue
 class ByteArrayReaderTest {
 
     @Test
+    @JsName("readUShortWithValueOf1AtAnOffset")
     fun `readUShort with value of 1, at an offset`() {
```

Comment:
I think I'd rather just have `fun readUShort_with_value_of_1_at_an_offset()` and no explicit `JsName`, and similar for the other tests. Having it named twice makes me worry about names getting out of sync in a future refactor. If you disagree feel free to ignore this.

Reply:
Sigh, that is a really good point. But the spaces in the name make it soooo clean. 😢 
I'm torn on which evil is better. @austinhobbs up to you how which approach you feel is best. 🤷

Reply:
For readability, I prefer the way it is currently. That said, the point around the names becoming out of sync is a good one. I think the snake_case function names could offer more benefit, even if a little ugly. I'll update the PR to move that direction.

Reply:
Removed the `@JsName` and updated function names.